training/distributed_training/pytorch/model_parallel/GPT2_huggingface_trainer/requirements.txt

```diff
@@ -0,0 +1,2 @@
+datasets>=1.8.0
+transformers>=4.21.1
```

Comment:
Change to `transformers==4.21.0` which is the latest supported version by smp.

Reply:
Ok sure! Would like to update to 4.21.1 later since it contains a bug fix.